src/Backups/BackupIO_S3.cpp

```diff
@@ -249,6 +249,24 @@ void BackupWriterS3::copyFileFromDisk(const String & path_in_backup, DiskPtr src
     BackupWriterDefault::copyFileFromDisk(path_in_backup, src_disk, src_path, copy_encrypted, start_pos, length);
 }
 
+void BackupWriterS3::copyFile(const String & destination, const String & source, size_t size)
+{
+    LOG_TRACE(log, "Copying file inside backup from {} to {} ", source, destination);
+    copyS3File(
+        client,
+        client,
```

Comment:
Broke my local build:

```
/home/ubuntu/repo/ClickHouse/src/Backups/BackupIO_S3.cpp:255:5: error: no matching function for call to 'copyS3File'
  255 |     copyS3File(
      |     ^~~~~~~~~~
/home/ubuntu/repo/ClickHouse/src/IO/S3/copyS3File.h:31:6: note: candidate function not viable: no known conversion from 'std::shared_ptr<S3::Client>' to 'const String' (aka 'const basic_string<char, char_traits<char>, allocator<char>>') for 2nd argument
   33 | void copyS3File(
      |      ^
   34 |     const std::shared_ptr<const S3::Client> & s3_client,
   35 |     const String & src_bucket,
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[10301/12493] Building CXX object src/CMakeFiles/dbms.dir/Functions/CastOverloadResolver.cpp.o
ninja: build stopped: subcommand failed.
```

Fix:

Reply:
Actually it's this PR https://github.com/ClickHouse/ClickHouse/pull/56314 which modifies `copyS3File`
It didn't have the latest master so it didn't catch the new code that calls `copyS3File`